src/main/java/org/apache/commons/collections4/ListUtils.java

```diff
@@ -539,7 +538,7 @@ public static int hashCodeForList(final Collection<?> list) {
                 }
             }
         }
-        return -1;
+        return ArrayUtils.INDEX_NOT_FOUND;
```

Comment:
Hi @arturobernalg 

I think you can replace `ArrayUtils.INDEX_NOT_FOUND` with `CollectionUtils.INDEX_NOT_FOUND` and delete `ArrayUtils.INDEX_NOT_FOUND` since it is package private. Just make sure you use a better comment for `CollectionUtils.INDEX_NOT_FOUND` like the one for `ArrayUtils.INDEX_NOT_FOUND`.

Reply:
Changed @garydgregory